test/test-app.js

```diff
@@ -49,6 +50,16 @@ describe('JHipster generator module', () => {
         it('generates default files', () => {
             assert.file(expectedFiles.module);
         });
+        it('generates index.js with correct content', () => {
+            const actualContent = fs.readFileSync('generators/app/index.js', 'utf8');
+            const expectedContent = fs.readFileSync(`${__dirname}/results/index.js.txt`, 'utf8');
```

Comment:
I'm a bit worried that these result files will need to be (manually?) updated for each minor change done in the generator.
Still, this is an interesting approach, that allows you to visualize the expected content.

@pascalgrimaud : what do you think?

Reply:
Same comment like you @murdos 
The files inside `results/` folder need to be updated for each changes, so I don't think it's a good idea to test the content

Reply:
In my opinion that is a good way to ensure that your expected generated content changes with your production code changes.

Currently there's no way to check that without actually running the generator against a prebuilt app.

I believe this automated test relieves one from having to do so every time there's a change.

Reply:
@vivekmore could you remove the result files and related tests from this PR? That will allow to merge this PR and the fixes it contains.

Reply:
Do you think you'll be able to update this PR soon? We need that to release a new version compatible with JHipster 6...

Reply:
Sorry busy at the moment.